.github/workflows/run-tests.yaml

```diff
@@ -0,0 +1,106 @@
+name: Run Tests
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+      source:
+        description: 'Source of tests'
+        required: true
+        type: string
```

Comment:
Shouldn't this be a `choice` as we have `options`?

Reply:
updated.

Reply:
you may forget to push the code to update this, it's not updated.

Reply:
Ah sorry about that. Fixed.